README.md

```diff
@@ -1,18 +1,27 @@
 # cortx-test
-CORTX-TEST is an automation repository for multiple automation projects developed for LDR R2 and future versions. 
-Right now, it can be divided into following logical loosely coupled parts 
-* Test framework. 
-* Test execution framework. 
-* Robot framework and 
+CORTX-TEST is a repository for multiple automation projects developed for CORTX (QSG Ref: https://github.com/Seagate/cortx/blob/main/README.md#get-started) and supported solutions/systems.
+
+It is logically divided into following components: 
+* Test Automation framework
+* Test Run Framework
+* Robot framework and
 * Tools (reporting, DI, clone TP, etc.,).
 
-## Start Here
-Make sure you brush up your Git knowledge if you are coming from svn or other versioning system. Create a GitHub account and a PAT, and get access to Seagate Repositories including Cortx-Test. Follow the link https://github.com/Seagate/cortx/blob/main/doc/github-process-readme.md to configure git on your local machine. Following Readme document will give you enough insights and start contributing.
+## Getting Started
+This document assumes that you are aware about Github and if you are coming from svn or other versioning system it is recommended to . Follow the link https://github.com/Seagate/cortx/blob/main/doc/github-process-readme.md to configure git on your local machine. Following Readme document will give you enough insights and start contributing.
+
+You can have a separate client VM with any Linux flavour to install client side pre-requisites and start using automation framework on the same VM. This VM should have connectivity to Cortx Cluster OR CORTX OVA deployment. Alternatively you may use one of the nodes as client (less recommended).     
```

Comment:
less or not recommended?

Reply:
less recommended. We should not ask to NOT do anything if someone wants to save on resources IMO!